pkg/executor/api.go

```diff
@@ -258,12 +314,19 @@ func (executor *Executor) GetHandler() http.Handler {
 	r.HandleFunc("/v2/tapServices", executor.tapServices).Methods("POST")
 	r.HandleFunc("/healthz", executor.healthHandler).Methods("GET")
 	r.HandleFunc("/v2/unTapService", executor.unTapService).Methods("POST")
+	r.HandleFunc("/v2/isValid", executor.isValidHandler).Methods("POST")
 	return r
 }
 
 // Serve starts an HTTP server.
 func (executor *Executor) Serve(ctx context.Context, port int) {
 	handler := otelUtils.GetHandlerWithOTEL(executor.GetHandler(), "fission-executor", otelUtils.UrlsToIgnore("/healthz"))
 	httpserver.StartServer(ctx, executor.logger, "executor", fmt.Sprintf("%d", port), handler)
-
 }
+
+type (
+	functionWithAddress struct {
+		SvcAddress string       `json:"svcAddress"`
+		Function   fv1.Function `json:"function"`
+	}
+)
```

Comment:
This type definition should be moved to wherever the other types for API request / return types are kept

Reply:
Changed it to types.go. Please take a look!